spec/sidekiq_helper.rb

```diff
@@ -17,6 +18,8 @@ def jid
 Sidekiq.configure_server(&configure_redis)
 Sidekiq.configure_client(&configure_redis)
 
+Sidekiq::Web.use Rack::Session::Cookie, :secret => SecureRandom.hex(32), :same_site => true, :max_age => 86_400
```

Comment:
Hey @ybiquitous 
I checked the CI output and noticed that it was pulling Sidekiq 6.3.1 which is odd as this gem is set up to support up to 6.1:

```
/home/runner/work/sidekiq-throttled/sidekiq-throttled/vendor/bundle/ruby/2.6.0/gems/sidekiq-6.3.1/lib/sidekiq/web.rb:154: warning: global variable `$TESTING' not initialized
```

https://github.com/ybiquitous/sidekiq-throttled/runs/4424654694?check_suite_focus=true

The tests pass locally so I went dig into the CI file and noticed that it has an outdated env name for the  `ruby/setup-ruby` action (`GEMFILE`), I believe if you rename the reference below to `BUNDLE_GEMFILE`, CI tests will pass:
https://github.com/sensortower/sidekiq-throttled/blob/7f562c34fdaa7ec1d2c0e5ebb7c2e5c4bf8573a8/.github/workflows/ci.yml#L28

Then I believe it's safe to remove this line (21) + line 6.

Reply:
@zrod Thank you and sorry for the late response!

I've added the Sidekiq version check like this:

```ruby
if Gem::Version.new(Sidekiq::VERSION) >= Gem::Version.new("6.2.0")
  ...
end
```